src/middleware.ts

```diff
@@ -21,5 +20,5 @@ export function middleware(request: NextRequest) {
 // Applies this middleware only to specific paths
 export const config = {
   matcher:
-    '/((?!api|static|_next|favicon\\.ico|.*\\.(?:png|jp?eg|gif|webp|svg|ico|riv)).*)',
+    '/((?!api|static|_next|favicon\\.ico|.*\\.(?:png|jp?eg|gif|webp|svg|ico|xml|txt|riv)).*)',
```

Comment:
@tcheee @oktapodia I have also added the latest change to our main branch to this ongoing task. It was helpful to test the sitemap and don´t think it does any harm to also have it in here. Wdyt?